Source/JavaScriptCore/parser/ParserModes.h

```diff
@@ -312,7 +312,8 @@ const LexicalScopeFeatures NoLexicalFeatures                           = 0;
 const LexicalScopeFeatures StrictModeLexicalFeature               = 1 << 0;
 
 const LexicalScopeFeatures AllLexicalFeatures = NoLexicalFeatures | StrictModeLexicalFeature;
-static_assert(AllLexicalFeatures <= 0b1111, "LexicalScopeFeatures must be 4bits");
+static constexpr unsigned bitWidthOfLexicalScopeFeatures = 3;
+static_assert(AllLexicalFeatures <= bitWidthOfLexicalScopeFeatures, "LexicalScopeFeatures must be 3bits");
```

Comment:
Ah, I think it should be `(1 << bitWidthXXX) - 1`

Reply:
oh lol oops